src/Table/Teradata/TeradataTableQueryBuilder.php

```diff
@@ -144,7 +145,8 @@ public function getCreateTableCommand(
 
         if ($primaryKeys !== []) {
             $columnsSql .= sprintf(
-                ",\nPRIMARY KEY (%s)",
+                ",\nCONSTRAINT %s PRIMARY KEY (%s)",
```

Comment:
```suggestion
                ',\nCONSTRAINT %s PRIMARY KEY (%s)',
```

Reply:
blbé, že je to interface to by mohl byt parametr ten kbc_pk :/

Reply:
tam musi byt `"` jinak se ten `\n` nevyhodnoti